### PR TITLE
Aws guardduty to py3

### PR DESCRIPTION
**Pull request by DinaMeylakh**
Converting AWSGD from python2 to python 3

### CODE REVIEW

File: Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py [hunk 1]
```diff
@@ -26,297 +21,249 @@ def default(self, obj):  # pylint: disable=E0202
...
-        ec = {
-            "AWS.GuardDuty.Detectors(val.DetectorId === obj.DetectorId).ThreatIntelSet"
-            "(val.ThreatIntelSetId === obj.ThreatIntelSetId)": data}
-        return create_entry('AWS GuardDuty ThreatIntel Set', data, ec)
-
-    except Exception as e:
-        return raise_error(e)
+def delete_detector(client: boto3.client, args: dict):
+    response = client.delete_detector(DetectorId=args.get('detectorId'))
+    if response == dict():
```
Comment (amshamah419) [inline Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py:37]: Why the change from this? `if response['ResponseMetadata']['HTTPStatusCode'] == 200:`

Comment (DinaMeylakh) [reply Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py:37]: When changing to python3 the boto3 client library changed to its py3 version
The new SDK responses do not include the 'ResonseMetadata' section anymore.
You can find documentation for the new responses structure in the following:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/guardduty.html#

**Review by amshamah419 — APPROVED** [review]
Nice job!
